src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```
@@ -1,18 +1,63 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
```

You can probably remove the extra TargetFrameworkOverride stuff with this:

```suggestion
    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.0;net472</TargetFrameworks>
    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
```

---

Either way feels like a hack; not sure one is better than the other.

---

The way that I outlined is the one that customers are usually using. I would prefer this offer the explicit override. If you directly build this one project on Unix with dotnet wouldn't that result in an error with net472 as the target framework not conditioned?

---

> If you directly build this one project on Unix with dotnet wouldn't that result in an error with net472 as the target framework not conditioned?

Yes. The error alerts users to the fact that `dotnet` on Linux cannot currently build a valid package for this project. I will add a comment to the TargetFrameworkOverride line to make it clear that it's only there because of dotnet/sdk#335. For the project file itself, the current form better captures the desired end state.

---

Comment now added